arlas-core/src/main/java/io/arlas/server/core/services/ExploreService.java

```diff
@@ -154,6 +161,23 @@ public void applyFilter(Filter filter, FluidSearchService fluidSearch) throws Ar
         }
     }
 
+    public void applyPartitionFilter(List<Filter> filters, FluidSearchService fluidSearch) throws ArlasException {
+        if (filters != null) {
+            for(Filter f:filters){
+                if (f != null) {
+                    CheckParams.checkFilter(f);
+                    if (f.f != null && !f.f.isEmpty()) {
+                        CollectionReference collectionReference = fluidSearch.getCollectionReference();
+                        if (!filterFHasDateQuery(f, collectionReference) && !StringUtil.isNullOrEmpty(f.dateformat)) {
+                            throw new BadRequestException("dateformat is specified but no date field is queried in f filter (gt, lt, gte, lte or range operations)");
```

Comment:
I find it harsh to raise an exception because the date format is provided and no date is provided. I believe the request is still valid when you add additional unused information.

Reply:
This part of code already existed and some IT test specifcally this comportement. I agree to change it but not in this PR.